src/Dictionaries/TrieDictionary.cpp

```diff
@@ -331,13 +480,136 @@ void TrieDictionary::loadData()
             {
                 const auto & attribute_column = *attribute_column_ptrs[attribute_idx];
                 auto & attribute = attributes[attribute_idx];
-                setAttributeValue(attribute, key_column->getDataAt(row_idx), attribute_column[row_idx]);
+
+                setAttributeValue(attribute, attribute_column[row]);
+            }
+
+            size_t row_number = ip_records.size();
+
+            std::string addr_str(key_column->getDataAt(row).toString());
```

Comment:
Extra string copy
Probably, `std::string_view` will work?

Reply:
I cahnge it to `string_view`, but `Poco::Net::IPaddress` accept `const std::string &`, and I convert to string anyway, is it ok?

Reply:
It is ok. At least it will be copied only for Poco